.github/workflows/main.yml

```diff
@@ -32,4 +32,5 @@ jobs:
     - run: flutter upgrade
     - run: flutter clean
     - run: flutter pub get
+    - run: flutter analyze
```

Comment:
@SoundSonic1 what does flutter analyze do and why is it needed in CI?

Reply:
It does static code analysis. It complains for example if variables are unused

Reply:
Cool, yeah that’ll be useful to know if CI fails or complains